storybook/index.html

```diff
@@ -386,6 +386,28 @@ <h1 class="sectionHeading">BAMP</h1>
                         </a>
                     </div>
                 </section>
+                <section class="demo" data-module-name="muralModal">
+                    <div class="muralModal">
+                        <img class="muralModal--img" src="https://greatschoolvoices.org/wp-content/uploads/2020/06/Lodestar1.jpeg">
+                        <div class="muralModal--container">
+                            <button class="muralModal--close">✖	</button>
+                            <div class="muralModal--name">Lodestar</div>
+                            <p class="muralModal--text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis nunc sed ipsum faucibus finibus. Donec sed efficitur lectus. Fusce risus tellus, sodales vel tellus convallis, aliquam ullamcorper velit. Vestibulum commodo velit ac eleifend tincidunt. Quisque tincidunt purus sem, at interdum dolor hendrerit eu. Quisque dui metus, laoreet at malesuada fringilla, ullamcorper tempus mauris.</p>
+                            <p>
```

Comment:
Is there a reason for the `<p>` tag here? I'm seeing this just for the artist and not for the location

Reply:
@kingtrentiii sorry, I missed this when I reviewed...

@coramonokandilos it looks like it's to introduce a line break before the "Location" line, while also adding some whitespace between them (I think the browser default `p` has some amount of margin)

Best thing is to add a class for this, and avoid depending on a default that we haven't reset in our `base` file, since it could vary between browsers.

But @kingtrentiii for now I think it'd be fine to just add another `<p>` around the location line -- if we have it on one we should have it on the other.